docs/root/start/sandboxes/grpc-s2s.rst

```diff
@@ -0,0 +1,229 @@
+.. _install_sandboxes_grpc_s2s:
+
+gRPC Service to Service
```

Comment:
Would you mind thinking about gRPC communication ways such as
```
1. Simple RPC
2. Server-to-client streaming RPC
3. Client-to-server streaming RPC
4. Bidirectional streaming RPC
```
Then it can make this sandbox more general.